third_party/openthread/src/ncp/spinel.h

```diff
@@ -51,16 +51,6 @@
 #define SPINEL_API_WARN_UNUSED_RESULT __attribute__((warn_unused_result))
 #endif // ifdef __GNUC__
 
-#if !defined(__BEGIN_DECLS) || !defined(__END_DECLS)
-#if defined(__cplusplus)
-#define __BEGIN_DECLS extern "C" {
-#define __END_DECLS }
-#else // if defined(__cplusplus)
-#define __BEGIN_DECLS
-#define __END_DECLS
-#endif // else defined(__cplusplus)
-#endif // if !defined(__BEGIN_DECLS) || !defined(__END_DECLS)
-
```

Comment:
Can you please help submit a PR in OpenThread with the same change to `spinel.h`?
`spinel.h` and `spinel.c` are maintained in OpenThread repo. In `wpantund` a copy is kept in `third_party` folder.